app/services/edition_unwithdrawer.rb

```diff
@@ -34,10 +34,18 @@ def fire_transition!
     unwithdrawn_edition.editorial_remarks << EditorialRemark.create(author: user, edition: edition, body: "Unwithdrawn")
 
     Edition::AuditTrail.acting_as(user) do
```

Comment:
👍 @gpeng . I just noticed this line was removed in the previous commit. I re-added it now. I think we don't want to change the original behaviour, don't we?